### PR TITLE
chore, feat, refactor(Egen): Clean up steps, template update etc.

### PR DESCRIPTION
**Pull request by krishr2d2**
<br>
The notebook `automl_forecasting_on_vertex_pipelines.ipynb` doesn't clean up the pipeline jobs and other resources created during while running the pipeline jobs. This PR includes a clean up step that deletes the pipeline jobs and resources like Vertex AI Dataset, batch prediction job and the uploaded models.

This PR includes the following updates:

- Replace REGION with LOCATION
- remove IS_TESTING in cleaning up section
- Add clean up steps for pipeline jobs, batch prediction job, uploaded models and Vertex AI Dataset
- Adds collab enterprise logo
- Makes structural changes according to new notebook template
- Removes the use of future tense
- Fixes typo 100MM to 100M (100 million)
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb [hunk 1]
```diff
@@ -102,9 +97,9 @@
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn how to create AutoML Forecasting models using [Vertex AI Pipelines](https://cloud.google.com/vertex-ai/docs/pipelines/introduction) downloaded from [Google Cloud Pipeline Components](https://cloud.google.com/vertex-ai/docs/pipelines/components-introduction) (GCPC). These pipelines are Vertex AI Tabular Workflow pipelines that are maintained by Google. These pipelines showcases different ways to customize the Vertex AI Tabular training process.\n",
+        "In this tutorial, you learn how to create AutoML Forecasting models using [Vertex AI Pipelines](https://cloud.google.com/vertex-ai/docs/pipelines/introduction) downloaded from [Google Cloud Pipeline Components](https://cloud.google.com/vertex-ai/docs/pipelines/components-introduction) (GCPC). These pipelines are Vertex AI Tabular Workflow pipelines that are maintained by Google. These pipelines showcase different ways to customize the Vertex AI Tabular training process.\n",
```
Comment (kittyabs) [inline notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb:100]: - change: "AutoML Forecasting"
- to: "AutoML forecasting"

File: notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb [hunk 2]
```diff
@@ -192,182 +198,111 @@
...
+        "id": "zebLBGXOky2A"
       },
       "source": [
-        "**1. Vertex AI Workbench**\n",
-        "* Do nothing since you're already authenticated."
+        "### Notes about service account and permission\n",
+        "\n",
+        "For full details of the permission setup, refer to [Service accounts for Tabular Workflows](https://cloud.google.com/vertex-ai/docs/tabular-data/tabular-workflows/service-accounts).\n",
+        "\n",
+        "**By default no configuration is required**, if you run into any permission related issue, please make sure the service accounts below have the required roles:\n",
```
Comment (kittyabs) [inline notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb:260]: remove "please"

File: notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb [hunk 3]
```diff
@@ -1425,8 +1357,8 @@
         "    training_fraction=training_fraction,\n",
         "    validation_fraction=validation_fraction,\n",
         "    test_fraction=test_fraction,\n",
-        "    # Please note that TFT model will ONLY ensemble the model from\n",
-        "    # the top one trial, so `num_selected_trials` can not be set for TFT model.\n",
+        "    # Please note that TFT model ONLY ensembles the model from\n",
```
Comment (kittyabs) [inline notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb:1360]: remove "Please"

File: notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb [hunk 4]
```diff
@@ -1658,9 +1582,9 @@
         "id": "8Bu0wywvPYkD"
       },
       "source": [
-        "## Integrate Tabular Workflow for Forecasting into your existing KFP pipeline\n",
+        "## Integrate Tabular Workflow for Forecasting with your existing KFP pipeline\n",
         "\n",
-        "This is implemented using [the pipeline-as-component feature](https://www.kubeflow.org/docs/components/pipelines/v2/load-and-share-components/) of KFP."
+        "In this section, you define and run a sample KFP pipeline with your Tabular Workflow pipeline integrated as a component. For this, you use [the pipeline-as-component feature](https://www.kubeflow.org/docs/components/pipelines/v2/load-and-share-components/) of KFP."
```
Comment (kittyabs) [inline notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb:1587]: - change: "For this, you use ..."
- to: "For this, use ..."

**Review by kittyabs — COMMENTED** [review]
several edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**